.travis.yml

```diff
@@ -6,7 +6,26 @@ env:
   - TOXENV=py33
   - TOXENV=py34
   - TOXENV=pypy
+  - TOXENV=flake8
 install:
   - travis_retry pip install tox
 script:
   - tox
+
+after_script:
+  - |
+      env | sort
+      if [ "$TOXENV" == "flake8" ]; then
+          return
+      fi
```

Comment:
We could also test for the presence of a `coverage.xml` file, that would scale better.